multitenancy/tenant-manager-api/src/main/java/io/apicurio/multitenant/api/TenantsResourceImpl.java

```diff
@@ -54,6 +57,7 @@
     }
 
     @Override
+    @Transactional
```

Comment:
I've seen you removed this annotation from the storage and added it here, is that really needed? maybe just with the changes to the entities annotations is enough?

Reply:
I think it is needed, yes.  If you have the `@Transactional` boundary on the storage, then the Tx is committed when the method returns.  This is fine for creates and gets, but not for updates.  Because for updates we call `findByTenantId` and then `save`.  But the save will fail because it's trying to persist an entity from another hibernate transaction/connection.  I don't have the specific exception anymore, but could reproduce it if you're interested.

I think if I used the JPA/Hibernate `merge` method on the entity manager directly prior to calling `persist`, it might work then.  I didn't try that, but it seems hacky to directly use the entity manager when using Panache.

Reply:
great explanation, thank you, I'm fine with it now :)